### PR TITLE
Add instructions for manual testing

### PR DESCRIPTION
**Pull request by ptrfrncsmrph**
I found this useful, I can't tell if those are the best commands to run but they were my best guess 🙂

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,6 +43,15 @@ Follow these instructions for contributing new recipes. The Goal headers indicat
     - If you do install `npm` dependencies for your recipe, please state which libraries were installed in the recipe's `README.md` file.
 1. Implement your recipe. If you add any new modules, always start the module name with your recipe's "Unique Recipe Name" (e.g. `MyNewRecipe.Foo`, `MyNewRecipe.Module.Path.To.Cool.Types`)
     - Run `spago -x recipes/MyNewRecipe/spago.dhall build -w` while in the root folder for faster iteration while developing
+1. Confirm your recipe behaves as expected:
```
Comment (JordanMartinez) [inline CONTRIBUTING.md:46]: You should just use `make RecipeName-web`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,6 +43,15 @@ Follow these instructions for contributing new recipes. The Goal headers indicat
...
+    - **For recipes that target the browser**, from the root folder run:
+        ```shell
+        npx parcel recipes/MyNewRecipe/web/index.html
+        ```
+    - **For recipes that target Node**, from the root folder run:
+        ```shell
+        npx spago -x recipes/MyNewRecipe/spago.dhall run -m MyNewRecipe.Main
```
Comment (JordanMartinez) [inline CONTRIBUTING.md:53]: You should just use `make RecipeName-node`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,6 +43,15 @@ Follow these instructions for contributing new recipes. The Goal headers indicat
     - If you do install `npm` dependencies for your recipe, please state which libraries were installed in the recipe's `README.md` file.
 1. Implement your recipe. If you add any new modules, always start the module name with your recipe's "Unique Recipe Name" (e.g. `MyNewRecipe.Foo`, `MyNewRecipe.Module.Path.To.Cool.Types`)
     - Run `spago -x recipes/MyNewRecipe/spago.dhall build -w` while in the root folder for faster iteration while developing
+1. Confirm your recipe behaves as expected:
+    - **For recipes that target the browser**, from the root folder run:
+        ```shell
+        npx parcel recipes/MyNewRecipe/web/index.html
```
Comment (JordanMartinez) [inline CONTRIBUTING.md:49]: You should just use `make RecipeName-web`

File: CONTRIBUTING.md (hunk 2 above)
Comment (ptrfrncsmrph) [reply CONTRIBUTING.md:53]: Aha 👍 I missed that, but it seems `make RecipeName-node` and `make RecipeName-web` both work, I hadn't seen that in the documentation and I'm not familiar enough with Makefiles to have parsed how that command should work. Is this already documented elsewhere, or is it still worthwhile adding the `make` commands to the docs here?

Comment (milesfrain) [reply CONTRIBUTING.md:53]: I believe this is the key section:
https://github.com/JordanMartinez/purescript-cookbook#running-recipes

But perhaps a small shell snippet showing how to run `HelloWorld` would be helpful. e.g.:
```
> make list           
Use "make RecipeName-target" to run a recipe

=== Node Recipes ===
make ReadPrintFileContents-node
make HelloWorld-node
...

=== Web Recipes ===
make WindowProperties-web
make HelloWorld-web
...

> make HelloWorld-node
=== Running HelloWorld on the Node.js backend ===
[info] Installation complete.
[info] Build succeeded.
Hello world!

> make HelloWorld-web 
=== Building HelloWorld ===
[info] Installation complete.
[info] Build succeeded.
=== Launching HelloWorld in the web browser ===
...
```

Feel free to put something like the above snippet in the main readme.

**Pull request closed without merging**